scripts/codex_autoreview.py

```
@@ -0,0 +1,409 @@
+        raw = resp.read()
+        if "application/json" in ctype:
+            return json.loads(raw.decode("utf-8"))
+        return raw
+
+
+def _get_event() -> dict[str, Any]:
+    event_path = os.environ.get("GITHUB_EVENT_PATH")
+    if not event_path:
+        print("GITHUB_EVENT_PATH not set; are we in GitHub Actions?", file=sys.stderr)
+        sys.exit(1)
+    with open(event_path, encoding="utf-8") as f:
+        data = json.load(f)
+    if not isinstance(data, dict):
+        print("Unexpected event payload type; expected object", file=sys.stderr)
+        sys.exit(1)
+    return cast(dict[str, Any], data)
+
+
+def _load_guidelines() -> str:
+    # Expected path in this repo
+    path = Path("prompts/code-review.md")
+    if not path.exists():
+        print("prompts/code-review.md not found", file=sys.stderr)
+        sys.exit(1)
+    return path.read_text(encoding="utf-8")
+
+
+def _list_changed_files(owner: str, repo: str, pr_number: int, token: str) -> list[ChangedFile]:
+    files: list[ChangedFile] = []
+    page = 1
+    per_page = 100
+    while True:
+        batch = _github_api(
+            "GET",
+            f"https://api.github.com/repos/{owner}/{repo}/pulls/{pr_number}/files",
+            token=token,
+            params={"page": page, "per_page": per_page},
+        )
+        if not batch:
+            break
+        for it in batch:
+            files.append(
+                ChangedFile(
+                    filename=it["filename"],
+                    sha=it["sha"],
+                    status=it.get("status", "modified"),
+                    patch=it.get("patch"),
+                )
+            )
+        if len(batch) < per_page:
+            break
+        page += 1
+    return files
+
+
+def _compose_prompt(guidelines: str, repo_root: Path, changed: list[ChangedFile], pr: dict) -> str:
+    # Include high-level PR context
+    pr_title = pr.get("title") or ""
+    pr.get("body") or ""
```

[P3] Unused expression discards PR body

In scripts/codex_autoreview.py, the line `pr.get("body") or ""` is a no-op (its result is not assigned or used). If the PR body was intended to be included in the prompt context, this silently drops it; otherwise it’s dead code and should be removed to avoid confusion.